packages/beacon-node/src/chain/seenCache/seenGossipBlockInput.ts

```diff
@@ -415,7 +415,7 @@ export class SeenGossipBlockInput {
     }
 
     // Return if we received all data columns while waiting for getBlobs
-    if (blockCache.cachedData && this.hasSampledDataColumns(blockCache.cachedData.dataColumnsCache)) {
+    if (this.hasSampledDataColumns(blockCache.cachedData.dataColumnsCache)) {
```

Comment:
I think this will throw without the `blockCache.cachedData` check if the block comes in first?